pkg/virt-handler/vm.go

```diff
@@ -2657,6 +2657,10 @@ func (d *VirtualMachineController) vmUpdateHelperDefault(origVMI *v1.VirtualMach
 			return fmt.Errorf("failed to adjust resources: %v", err)
 		}
 	} else if vmi.IsRunning() {
+		if err := d.hotplugSriovInterfaces(vmi); err != nil {
```

Comment:
Now that this appears in the "regular" reconcile flow (i.e. not part of the migration flow), what happens when this will run against an older virt-launcher where this command is not recognize?

Will this command return an error and we will just ignore it? Can you check this against a build that is missing the server side of this command?

Reply:
As far as I understand it will return error from the gRPC client that such command is not exists.
And since we don't fail on errors at the moment the loop will continue regularly, I will verify it as you suggest.

Reply:
@ormergi did you verify it?

Reply:
@AlonaKaplan  @EdDev I verified that in case of new virt-handler tries to send SR-IOV hotplug command it will fail and the following error message will appear in the logs:
```
{"component":"virt-handler","kind":"","level":"error","msg":"failed to hot-plug SR-IOV interfaces: unknown error encountered sending command HotplugHostDevices: rpc error: code = Unimplemented desc = unknown method HotplugHostDevices for service kubevirt.cmd.v1.Cmd","name":"sriovvmi1","namespace":"default","pos":"vm.go:2531","timestamp":"2022-01-12T15:44:51.175179Z","uid":"5deabbb2-e281-4053-9441-048be6589e74"}
```